### PR TITLE
Adjust consent properties

### LINKED ISSUE #213: Adjust registration properties for consents, grants, and receipts
- Consider replacing registeredBy/With/At/Agent with grantedBy, grantedWith, grantedAt, grantee in Access Consents, Data Consents, Access Grants, and Data Grants
- Remove fromSocialAgent from AccessReceipt
- Replace viaSocialAgent in AccessReceipt with grantedBy

### PR DESCRIPTION
**Pull request by justinwb**
Resolves #213

### CODE REVIEW

File: proposals/specification/access-consents.bs [hunk 1]
```diff
@@ -42,22 +42,22 @@ has been granted access. [=Access Grants=] are generated from
   </thead>
   <tbody>
     <tr>
-      <td>registeredBy</td>
+      <td>grantedBy</td>
       <td>[SocialAgent](#classSocialAgent)</td>
       <td>[=Social Agent=] that recorded the [=Access Consent=]</td>
```
Comment (elf-pavlik) [inline proposals/specification/access-consents.bs:47]:
```suggestion
      <td>[=Social Agent=] that granted the [=Access Consent=]</td>
```

File: proposals/specification/access-consents.bs [hunk 2]
```diff
@@ -42,22 +42,22 @@ has been granted access. [=Access Grants=] are generated from
...
     </tr>
     <tr>
-      <td>registeredWith</td>
+      <td>grantedWith</td>
       <td>[Application](#classApplication)</td>
       <td>[=Application=] used to record the [=Access Consent=]</td>
```
Comment (elf-pavlik) [inline proposals/specification/access-consents.bs:52]:
```suggestion
      <td>[=Application=] used to grant the [=Access Consent=]</td>
```

File: proposals/specification/access-consents.bs [hunk 3]
```diff
@@ -42,22 +42,22 @@ has been granted access. [=Access Grants=] are generated from
...
     </tr>
     <tr>
-      <td>registeredAt</td>
+      <td>grantedAt</td>
       <td>xsd:dateTime</td>
       <td>Date and time the [=Access Consent=] was created</td>
```
Comment (elf-pavlik) [inline proposals/specification/access-consents.bs:57]:
```suggestion
      <td>Date and time the [=Access Consent=] was granted</td>
```

File: proposals/specification/access-consents.bs [hunk 4]
```diff
@@ -253,17 +253,17 @@ needs to change, it must be replaced.
   </thead>
   <tbody>
     <tr>
-      <td>registeredBy</td>
+      <td>grantedBy</td>
       <td>[SocialAgent](#classSocialAgent)</td>
       <td>[=Social Agent=] that registered the [=Access Grant=]</td>
```
Comment (elf-pavlik) [inline proposals/specification/access-consents.bs:258]:
```suggestion
      <td>[=Social Agent=] that granted the [=Access Grant=]</td>
```

File: proposals/specification/access-consents.bs [hunk 5]
```diff
@@ -253,17 +253,17 @@ needs to change, it must be replaced.
...
     </tr>
     <tr>
-      <td>registeredAt</td>
+      <td>grantedAt</td>
       <td>xsd:dateTime</td>
       <td>Date and time the [=Access Grant=] was created</td>
```
Comment (elf-pavlik) [inline proposals/specification/access-consents.bs:263]:
```suggestion
      <td>Date and time the [=Access Grant=] was granted</td>
```

File: proposals/specification/access-consents.bs [hunk 6]
```diff
@@ -253,17 +253,17 @@ needs to change, it must be replaced.
...
     </tr>
     <tr>
-      <td>registeredAgent</td>
+      <td>grantee</td>
       <td>[Agent](#classAgent)</td>
       <td>Links to the [=Social Agent=] or [=Application=] that was granted access.</td>
```
Comment (elf-pavlik) [inline proposals/specification/access-consents.bs:268]:
```suggestion
      <td>The [=Social Agent=] or [=Application=] that was granted access.</td>
```

File: proposals/specification/access-consents.bs [hunk 7]
```diff
@@ -338,7 +338,7 @@ needs to change, it should be replaced.
       <td>[=Social Agent=] that owns the data being granted</td>
     </tr>
     <tr>
-      <td>registeredAgent</td>
+      <td>grantee</td>
       <td>[Agent](#classAgent)</td>
       <td>The [=Social Agent=] or [=Application=] that has received
       consent</td>
```
Comment (elf-pavlik) [inline proposals/specification/access-consents.bs:344]:
```suggestion
      <td>The [=Social Agent=] or [=Application=] that was granted access.</td>
```

File: proposals/specification/interop.ttl [hunk 8]
```diff
@@ -274,6 +274,38 @@ rdfs:label "Terms for application interoperability in Solid"@en .
   rdfs:range :DataConsent ;
   rdfs:label "Inherits From Consent"@en .
 
+:grantedBy
+  a rdf:Property ;
+  rdfs:comment "Social agent that issued consent"@en ;
```
Comment (elf-pavlik) [inline proposals/specification/interop.ttl:279]:
```suggestion
  rdfs:comment "Social agent that granted consent or grant"@en ;
```

File: proposals/specification/interop.ttl [hunk 9]
```diff
@@ -274,6 +274,38 @@ rdfs:label "Terms for application interoperability in Solid"@en .
...
+  rdfs:isDefinedBy <> ;
+  rdfs:domain [ owl:unionOf ( :AccessConsent :AccessGrant :AccessReceipt ) ] ;
+  rdfs:range :SocialAgent ;
+  rdfs:label "Granted By"@en .
+
+:grantedWith
+  a rdf:Property ;
+  rdfs:comment "Application used by a Social Agent to issue consent"@en ;
```
Comment (elf-pavlik) [inline proposals/specification/interop.ttl:287]:
```suggestion
  rdfs:comment "Application used to grant consent or grant"@en ;
```

File: proposals/specification/interop.ttl [hunk 10]
```diff
@@ -274,6 +274,38 @@ rdfs:label "Terms for application interoperability in Solid"@en .
...
+  rdfs:isDefinedBy <> ;
+  rdfs:domain :AccessConsent ;
+  rdfs:range :Application ;
+  rdfs:label "Granted With"@en .
+
+:grantedAt
+  a rdf:Property ;
+  rdfs:comment "Time that consent was granted"@en ;
```
Comment (elf-pavlik) [inline proposals/specification/interop.ttl:295]:
```suggestion
  rdfs:comment "Time that consent or grant was granted"@en ;
```

File: proposals/specification/interop.ttl [hunk 11]
```diff
@@ -274,6 +274,38 @@ rdfs:label "Terms for application interoperability in Solid"@en .
...
+  rdfs:isDefinedBy <> ;
+  rdfs:domain [ owl:unionOf ( :AccessConsent :AccessGrant ) ] ;
+  rdfs:range xsd:dateTime ;
+  rdfs:label "Granted At"@en .
+
+:grantee
+  a rdf:Property ;
+  rdfs:comment "Recipient of consent"@en ;
```
Comment (elf-pavlik) [inline proposals/specification/interop.ttl:303]:
```suggestion
  rdfs:comment "Recipient of consent or grant"@en ;
```

**Review by elf-pavlik — APPROVED** [review]
I left some suggestions for aligning every comment/description of those properties. Please feel free to pick/discard/amend each of those suggestions.

File: proposals/specification/interop.ttl (hunk 9 above)
Comment (justinwb) [reply proposals/specification/interop.ttl:287]:
```suggestion
  rdfs:comment "Application used by a Social Agent to grant consent or issue access grant"@en ;
```

**Pull request merged**